usr/share/lib/ipa/tests/SLES/SAP/test_sles_sap_license.py

```diff
@@ -11,6 +11,7 @@ def test_sles_sap_license(
         pytest.skip('SLES12-SP4 has combined license.')
 
     license_dirs = [
+        '/etc/YaST2/licenses/base/',
```

Comment:
This overlaps with the base license test. https://github.com/SUSE-Enceladus/ipa/blob/7a3a37e655013743f125bb2798f19e427ce11b12/usr/share/lib/ipa/tests/SLES/test_sles_license.py#L6

If sp1 has a combined license we should update the skip case above. E.g. `if in [12-sp1, 12-sp4] pytest.skip`.

Reply:
Otherwise this will cause all other sles versions with a base license @ `/etc/YaST2/licenses/base/` to pass eagerly in SAP test and therefore not properly test the SAP license.

Reply:
So... should I just not run the `test_sles_sap_license` test on SLES for SAP 12 SP1 & SP2 ?

Reply:
You can, but the images I tested passed okay with the sap and regular license tests. Let me do some more testing.